jenkins_job_wrecker/modules/properties.py

```diff
@@ -23,6 +24,16 @@ def gen_yml(self, yml_parent, data):
             if object_name == 'parametersdefinitionproperty':
                 self.registry.dispatch(self.component, 'parameters', child, parameters)
                 continue
+            elif object_name == 'pipelinetriggersjobproperty':
+                # Pipeline scripts put triggers in properties section
+                # Could import triggers in this block but I like to keep them
+                #   at the top of the file for cleanliness reasons
```

Comment:
Great job keeping imports at the top :) :beers: 

In fact you can delete this comment